### PR TITLE
add note to event

### PR DESCRIPTION
**Pull request by houqp**

### CODE REVIEW

File: src/oncall/ui/static/js/incalendar.js [hunk 1]
```diff
@@ -1245,6 +1245,14 @@
               })
             )
           )
+          .append(
```
Comment (dwang159) [inline src/oncall/ui/static/js/incalendar.js]: Looks like you can add a note, but it doesn't actually appear in the UI when viewing event details or editing.

File: src/oncall/ui/static/js/incalendar.js [hunk 2]
```diff
@@ -1245,6 +1245,14 @@
...
+            $('<li />')
+            .append('<label class="label-col">Note: </label>')
+            .append(
+              $('<div class="input-col" />')
+              .append('<input type="text" class="typeahead" id="inc-event-note" name="inc-event-note" value="" style="width:100%" /> ')
```
Comment (dwang159) [inline src/oncall/ui/static/js/incalendar.js]: Giving this the typeahead class makes it spawn a user typeahead

File: src/oncall/ui/static/js/incalendar.js (hunk 1 above)
Comment (houqp) [inline src/oncall/ui/static/js/incalendar.js]: Yeah, not enough time to finish that part on a train :P

**Comment by houqp** [comment]
Should be ready now.

**Comment by houqp** [comment]
updated, missed the editing view :(

**Pull request merged**